Sources/Web3Core/Utility/Array+Extension.swift

```diff
@@ -106,3 +106,12 @@ extension Array where Element: BinaryInteger {
         return sorted_data[index]
     }
 }
+
+// MARK: - Conversion
+
+/// Transforms `[Any?]` into `[AnyObject]`
+extension Array where Element == Any? {
+    func toAnyObject() -> [AnyObject] {
+        self.map { $0 as AnyObject }
+    }
```

Comment:
This could fail, since Any is more broad that AnyObject is, so the casting could fail.

Reply:
I do not see how this can cause problems. @yaroslavyaroslav  Could you please give more explanation/examples?

It's said that all classes implicitly conform to `AnyObject` but we're able to cast structs as well so maybe that part about structs (or to be specific `value types` is confusing).

--------

Done some research and found that casting value types to `AnyObject` produces a wrapper of type `__SwiftValue`. And here is the [`__SwiftValue`](https://github.com/apple/swift/blob/main/stdlib/public/runtime/SwiftValue.h#L13). Looks at the docs on line 13.

Reply:
Lol, why did they do that? Like now there's completely no difference between `Any` and `AnyObject` isn't it? If so, i agree that this would not fall, but i still suggest to make that extension more consistent, using either `Any` or `AnyObject` in whole method.

Reply:
There is still some difference when it comes to cast back to the original type. I cannot explain that in detail but for example if we cast `UInt64(1)` to `AnyObject` and try to use it as `if let bool = uint64AsAnyObject as? Bool` it will successfully cast to `Bool`. But if you cast it to `Any` instead it will cast back only as `UInt64`.

Reply:
> but for example if we cast UInt64(1) to AnyObject and try to use it as if let bool = uint64AsAnyObject as? Bool it will successfully cast to Bool. But if you cast it to Any instead it will cast back only as UInt64.

Actually I'd consider this as a bug of those thing that is casting to AnyObject for Objective-C environment. But I don't sure though.